README.md

```diff
@@ -48,13 +48,22 @@ I'm not far from completing [milestone 0.1.0](https://github.com/buzz-language/b
 cd mir
 make
 ```
-4. Add to your shell rc:
-```bash
-export BUZZ_PATH="/path/to/buzz/dist"
+4. Have fun: `zig build run -- <myscript.buzz>`
+
+You can also install buzz locally.
+
+```
+# set PATH
+export BUZZ_PATH="path-to-this-repo/zig-out"
 export PATH="$BUZZ_PATH/bin:$PATH"
+
+# direct install
+zig build -Doptimize=ReleaseSafe install -p ~/.local # insall locally
```

Comment:
shouldn't BUZZ_PATH points to ~/.local/buzz then ?

Reply:
not needed

Reply:
Is it ok to remove BUZZ_PATH from public documentation? It's really not needed.